cmp/filter/FilterChooserFieldSpec.ts

```diff
@@ -14,40 +15,42 @@ import {renderToStaticMarkup} from 'react-dom/server';
 
 const {INT, NUMBER} = FieldType;
 
+export interface FilterChooserFieldSpecConfig extends BaseFilterFieldSpecConfig {
```

Comment:
That name is a bit of a mouthful -- not sure if anything to be done

Reply:
This is probably where projects start using the leading `I`...  I'm still in favor of our trailing `Config` convention, and think these occasionally long names (with e.g. `SpecConfig`) are just the price we pay for it.